### ISSUE TITLE
Hide GitHub oauth token from AWS console

### ISSUE THREAD

**Issue by jackwarner** [manwaring/frigg#40]
![2018-01-04_1550](https://user-images.githubusercontent.com/244898/34583896-7fa37e94-f167-11e7-884a-f56f059a043c.png)

Ideally, don't ever echo out github tokens to the console.

Even more ideally, don't give anyone the ability to log into the console :)

**Comment by manwaring** [comment]
Are you using an older version of frigg?  The parameter should be stored in SSM and only retrieved at runtime - and never logged at runtime (and certainly never logged at stack creation time)

Here's the relevant snippet from utils/parameters.js:

```
module.exports.getGitHubAccessToken = () => {
  return getSSMParameter('/GITHUB_TOKEN/frigg-access')
    .then(data => data.Parameter.Value);
}
```

**Comment by jackwarner** [comment]
Yes, I was using an older version. I retried the deploy and GITHUB_TOKEN no longer shows up, but GITHUB_WEBHOOK_SECRET still does (below). I haven't done enough digging yet to see what kind of things you can do with that parameter alone.
